### PR TITLE
update media links to make it work with latex / pdf renderer

### PR DESCRIPTION
**Pull request by robert-zaremba**

### CODE REVIEW

File: notebooks/15 Pipelining Estimators.ipynb [hunk 1]
```diff
@@ -160,7 +160,7 @@
     "This will then be fed to the ``fit`` of the next step, and so on.\n",
     "Finally, on the last step, only ``fit`` is called.\n",
     "\n",
-    "![pipeline](figures/pipeline.svg)\n",
```
Comment (amueller) [inline notebooks/15 Pipelining Estimators.ipynb:163]: Then it doesn't work in the browser any more...

Comment (amueller) [reply notebooks/15 Pipelining Estimators.ipynb:163]: nbconvert should be able to convert svg to pdf for latex compilation. If not, that's an issue in nbconvert - though thinking about it, there might have been issues last time I tried. Maybe that's only true for python generated figures, not embedded ones. I think the only way to make this render in both is to use PNG :-/

Comment (robert-zaremba) [reply notebooks/15 Pipelining Estimators.ipynb:163]: ah, right. Then I guess it's better to leave this changeset

**Comment by robert-zaremba** [comment]
png is not rendered in the browser: https://github.com/amueller/scipy-2016-sklearn/pull/84#discussion_r99842325

**Comment by amueller** [comment]
Sorry I don't understand the last comment. PNG is rendered in the browser and in latex, PDF is not rendered in the brower, SVG is not rendered in latex.
Converting all SVG to PNG is a quick work-around - though I feel like there should be a way to make latex accept SVG, but I remember it was tricky.

**Comment by rasbt** [comment]
Hm, it may be different now, but I think you'd need external tools (like inkscape) to export/prepSVGs so that they can be embedded in latex. I remember struggling with that one once. Since then, I am just using eps or PDF for latex specifically, and PNG for everything else.

**Comment by robert-zaremba** [comment]
@rasbt  - correct, I had problems with svg.
@amueller - with PNG you loose quality.

**Comment by amueller** [comment]
@robert-zaremba I know, it's a horrible workaround to convert vector graphics to raster images, but I don't know another way without using the inkscape conversion.

**Comment by amueller** [comment]
One way is actually to render it using maplotlib because then you can embed multiple mime-types but that seems like a weird thing to do...

**Comment by rasbt** [comment]
> with PNG you loose quality.

yeah, that's the unfortunate compromise. However, I think for typical web media, it's probably more important to have it compatible for most people (and one can save the vector graphics to generate higher res versions later if needed, i.e., if everyone has 4k monitors one day or so :P)

**Pull request closed without merging**